react-native-app/services/AuthService.ts

```diff
@@ -20,16 +20,33 @@ const AuthService = {
 		return await this.attemptLogin(email, password);
 	},
 
-	async attemptLogin(email, password): Promise<User | undefined> {
-		let userCognitoData = await Auth.signIn(email, password);
-		if (!userCognitoData) {
-			throw new Error("Failed to authenticate with given credentials");
+	async attemptSignUp(data:any):Promise<string> {
+		try {
+			return await ServerFacade.signUp(data);
+		} catch {
+			throw new Error("Error on sign up. Please try again.");
+		}
+
+	},
+
+	async attemptLogin(email:string, password:string): Promise<User | undefined> {
+		const data = {
+			username: email,
+			password
+		}
+
+		let userCognitoData:any = await ServerFacade.login(data);
```

Comment:
This has been completed. I used the user return on the login instead of having to call getUser function to get user... again. So this should be fixed? let me know.